.changeset/odd-nails-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"@devopness/ui-react": minor
+---
+
+Fixed a bug in Dropdown to inoperate disabled options
```

Comment:
```suggestion
Fixes Dropdown closing after clicking a disabled option
```

I believe the description can be rewritten in simpler terms

Can you apply the suggestion?

Reply:
applied suggestion.